addons/custom-project-player/userscript.js

```diff
@@ -0,0 +1,23 @@
+export default async function ({ addon, global, console }) {
+  var id = window.location.href;
+  var pid = id.match(/\d/g);
+  pid = pid.join("");
+
+  while (1) {
+    let player = await addon.tab.waitForElement(".guiPlayer", { markAsSeen: true });
```

Comment:
you set `player` to a variable but never use it? Use it on line 18.

Reply:
well if i just replace player the layout of the page gets wonky so i just put the iframe inside of guiplayer

Reply:
No I meant instead of the `document.querySelector(".guiPlayer")` part